Project-Bootcamp/querys/query.js

```diff
@@ -0,0 +1,66 @@
+var $ = require ('jquery');
+var _ = require ('lodash');
+
+var Apikey = 'qrmacznjya3vdnynqdr4trf3bd9t3fau';
+var Locale = 'en_US';
+
+var BlizzardAPI = function () {
+
+    this.apikey = Apikey;
+    this.locale = Locale;
+};
+
+BlizzardAPI.prototype.RealmStatus = function (callback) {
+
+    $.ajax({
+        url: 'https://us.api.battle.net/wow/realm/status?locale=' + this.locale + '&apikey=' + this.apikey,
+        type: 'get',
+        datatype: 'json',
+        success: function (response) {
+            callback (response.realms);
+        }
+    })
+};
+
+BlizzardAPI.prototype.CharacterInfo = function (realm, name, callback) {
+
+    $.ajax({
+        url: 'https://us.api.battle.net/wow/character/' + this.realm + '/' + this.name
+                                                        + '?fields=stats%2C+items%2C+pvp%2C+guild'
+                                                        + '&locale=' + this.locale + '&apikey=' + this.apikey,
+        type: 'get',
+        datatype: 'json',
+        success: function (response) {
+            console.log(response);
+            callback (response);
```

Comment:
You should check that ``callback`` is defined (also, remove extra space before ``(``).